### DISCUSSION TITLE
Is KeepassXC challenged by the Same Secuirty Questions as Keepass?

### DISCUSSION THREAD

**Discussion by asjones987** [keepassxreboot/keepassxc#9072] [Q&A] [answered]
On the thread 
"someone can read the passwords using export trigger"
https://sourceforge.net/p/keepass/discussion/329220/thread/a146e5cf6b/

There are questions around a possible security issue with Keepass. Some say it is an issue. The author says anything could have this issue.... My thought is maybe there are things that can be done to mitigate the concern.

Does KeepassXC have this same issue or concern? If not why not?

thakns

Alan

**Comment by droidmonkey** [comment, marked as answer]
We don't support any triggers, we are not vulnerable.
